website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,273 @@
+
+### Default Data Sources
+
+The default data source for each function (where multiple sources are available) can be defined within the [`/sources` menu](https://docs.openbb.co/terminal/usage/guides/changing-sources).  The available sources for each function are displayed on the right of the menu, and they can be distinguished by the square brackets and distinct font color group.  Unless a preference for a particular function is defined, the command will prioritize in the order they are displayed, from left-to-right, on the Terminal screen.  To override a preference or default source, select one of the other choices by attaching the, `--source`, argument to the command syntax.  The available sources for the feature will be populated by auto complete when the `space bar` is pressed after typing `--source`.  This information is also printed with the `--help` dialogue of a command.
+
+### Importing and Exporting Data
+
+Most functions provide a method for exporting the raw data as a CSV, JSON, or XLSX file.  Exported data and user-supplied files to import are saved to the [OpenBBUserData folder](https://docs.openbb.co/terminal/usage/guides/data).  The folder is located at the root of the operating system's User Account folder.  Follow the link for a detailed description.
+
+## Charts
 
-The default values you see within `load -h` have been inputted here. E.g. the starting period is 2020-02-11. I can decide to change these default values by calling the argument and inputting a different value.
+The OpenBB charting library has themes for light and dark mode.
 
-Whenever you wish to apply an optional argument, you use the related shortcode, e.g. `-s` or `--start`. Then, if there is an additional word behind the argument (in this case there is, which is `START`) it implies the argument expects you to define a value. Within the documentation you can read that the format must be `YYYY-MM-DD` implying that `2010-01-01` will be valid. If there is not an additional word behind it, it is enough to write down `load AMZN -p` (which refers to the prepost optional argument)
+### Light and Dark Mode
 
-Let's change the starting and ending period of the data that is being loaded in by doing the following:
+Set the theme, globally, as either light or dark.
 
+Dark mode:
+
+```console
+/settings/chart -s dark
 ```
-(🦋) /stocks/ $ load AMZN -s 2005-01-01 -e 2010-01-01
 
-Loading Daily data for AMZN with starting period 2005-01-03.
+Light mode:
 
+```console
+/settings/chart -s light
 ```
 
-We can check that this period has changed by looking into the <a href="https://www.investopedia.com/trading/candlestick-charting-what-is-it/" target="_blank" rel="noreferrer noopener">candle chart</a> with `candle`. This, again shares the same `-h` argument. This results in the following which indeed depicts our selected period.
+The default state is dark mode.
 
+```console
+/stocks/load aapl -w/candle
```

Comment:
I would drop this because we can change it in the chart itself and changing a theme color isnt important enough in my mind to warrant being on the basics page.

Reply:
Sections on chart/table theme commands removed.